bp_common/src/include/bp_common_aviary_pkg.vh

```diff
@@ -15,7 +15,7 @@ package bp_common_aviary_pkg;
       ,num_lce: 1
 
       ,vaddr_width: 39
-      ,paddr_width: 39
+      ,paddr_width: 56
```

Comment:
39 is correct.  We only support 39 bits of physical address.  Please revert.

Reply:
Can't we just leave the address at 56 bits according to match the spec and just use the lower 39 bits of it?

Reply:
No, this is the parameter which defines all of our internal structs, which are 39 bits.

Reply:
Anywhere you're reading from software (your PTW) should use the 56 bit rv64 definition and truncate to 39